src/ui/pages/usersList/UsersList.css

```diff
@@ -0,0 +1,24 @@
+.users-list {
+  margin: 0 auto;
+  padding: 30px 0;
+  width: calc(100% - 32px);
+  max-width: 829px;  
```

Comment:
Where did this number come from?

Reply:
![image](https://user-images.githubusercontent.com/110392744/183016654-c02d9d6a-0e12-482c-b19c-7c9b7cc36e9c.png)
The Zeplin says its 829px

Reply:
Its a bad idea to use numbers directly from Zeplin like this, thats just the width of the frame used in the designs. Just use a percentage of the screen width instead

Reply:
added rule width:90% if screen's width > 900px